modules/openapi-generator/src/main/resources/dart2/api_test.mustache

```diff
@@ -1,13 +1,14 @@
-import 'package:{{pubName}}/api.dart';
-import 'package:test/test.dart';
+{{>header}}
+import "package:{{{pubName}}}/api.dart";
+import "package:test/test.dart";
```

Comment:
Here again, I've seen `import '` (single quote), but this line has `"` - inconsistent style.